### ISSUE TITLE
Is segmentation working?

### ISSUE THREAD

**Issue by sethb3744** [spacetx/starfish#973]
Hi,

I read on the starfish documentation that starfish is still in its prototype form, but I wanted to confirm if just one part of the pipeline is functioning or not. I am only interested in using the segment command. Is this in working order or is it still a work in progress?

Thanks,
Seth

**Comment by ttung** [comment]
Segmentation has one algorithm implemented (watershed segmenter).  Each stage of the pipeline can have multiple algorithm implementations.

**Comment by neuromusic** [comment]
@sethb3744 I'm curious... are you looking to do segmentation on image-based transcriptomics data? or some other imaging data?

**Comment by sethb3744** [comment]
@neuromusic Yes, it's for image-based transcriptomics data. My lab partner performed RNAscope, an in situ hybridization assay for detection of target RNA in intact cells. I'll attach examples of the TIF files produced below.
RFP: https://s3-us-west-1.amazonaws.com/shen-sbergenholtz/Enh1-3D-2.lif+-+Series001+-+C%3D3.tif
DAPI: https://s3-us-west-1.amazonaws.com/shen-sbergenholtz/Enh1-3D-2.lif+-+Series001+-+C%3D20.tif
GFP: https://s3-us-west-1.amazonaws.com/shen-sbergenholtz/Enh1-3D-2.lif+-+Series001+-+C%3D2.tif

**Comment by sethb3744** [comment]
@ttung Sorry, that's what I meant. I am interested in using Watershed, but I was asking to see if segment was in working condition. Does that mean that segment/Watershed is fully functional? I just wanted to confirm this before I moved forward with using starfish, since I know it's still a work in progress.

**Comment by ttung** [comment]
Yes it should be functional.

**Comment by ambrosejcarr** [comment]
@sethb3744 It's prudent of you to check. :-) If it's helpful to see an analysis prior to digging in on your own data, you can find a worked example that uses watershed here: 

[https://github.com/spacetx/starfish/blob/master/notebooks/ISS_Pipeline_-_Breast_-_1_FOV.ipynb](https://github.com/spacetx/starfish/blob/master/notebooks/ISS_Pipeline_-_Breast_-_1_FOV.ipynb)

**Comment by sethb3744** [comment]
Thanks all for your help! I got Watershed to work today with my own formatting Python script. @ambrosejcarr your link was very helpful, as was the formatting vignette (https://spacetx-starfish.readthedocs.io/en/latest/usage/iss/iss_vignette.html). Thanks again!
